.github/workflows/dependencies-head.yml

```diff
@@ -17,6 +17,8 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
@kratsg I'll look at this to help debug it, but this isn't needed for this workflow. You only need this if you need to get version information.

Reply:
@kratsg 

Ah, I understand why these need to have the full fetch depth now given that

https://github.com/scikit-hep/pyhf/blob/546022254a526ea62af8cbc6594ec881cddfa9f1/tests/test_public_api.py#L11

makes use of the version information.

Feel free to correct me if I'm wrong, but I think that the _only_ places that the CI needs to have the full fetch depth and tags are where the version number is going to get checked (so like `test_public_api.py`). So this means that these `fetch-depth: 0`s could get removed after `v0.6.2` is out, right? If you agree can you make this comment of mine an Issue, assign it to me, and put it on the `v0.7.0` board?

Reply:
This is now Issue #1465